clients/Android/ultrablue/app/src/main/java/fr/gouv/ssi/ultrablue/fragments/ProtocolFragment.kt

```diff
@@ -44,7 +39,7 @@ val ultrablueChrUUID: UUID = UUID.fromString("ebee1790-50b3-4943-8396-16c0b7231c
     The protocol itself is defined in the UltrablueProtocol.kt file.
  */
 class ProtocolFragment : Fragment() {
-    private var state = State.ENROLLMENT
```

Comment:
I won't fight for it, but in general it's a good idea to use enumerations with meaningful names instead of booleans. Not sure why you changed it here.

https://abseil.io/tips/94 (written for C++ but applies equally well to Java)

Reply:
Good advice!

Reply:
In our case, I don't know it it is relevant, as this boolean is used inside the class function, and not passed to any exposed or external function.
I wrote it as an enum first because I thought the whole protocol state machine woud be written here, but I later decided to move it to the UltrablueProtocol class